config/routes.rb

```diff
@@ -23,7 +23,7 @@
     resources :innings, only: [:show]
     resources :zoom_aliases, only: [:update]
     resources :turing_modules, path: '/modules', only: [:show, :create], shallow: true do
-      resources :attendances, only: [:create, :show, :edit, :update] do
+      resources :attendances do
```

Comment:
Let's avoid opening up all attendance routes. I believe you only need :destroy for this feature

Reply:
Changed attendance resources back to only: [:create, :show, :edit, :update] and added :destroy